topsort-api-v2.yml

```diff
@@ -217,13 +217,13 @@ components:
   schemas:
     Category:
       type: object
-      description: An object describing the category of the bids that will participate in an auction.
+      description: An object describing a category of the bids that will participate in an auction.
       required:
         - id
       properties:
         id:
           type: string
-          description: The marketplace's ID of the category for which an auction will be run.
+          description: A marketplace's ID of one the categories for which an auction will be run.
```

Comment:
This description should say `one of the`. However, I think we are repeating the auction information in 3 different places. First in the Auction model, then in the Category description and finally here in the id.